src/ops/element_binary.cc

```diff
@@ -108,9 +108,8 @@ Tensor FFModel::divide(const Tensor in1,
 }
 
 bool ElementBinaryParams::is_valid(
-    std::pair<ParallelTensorShape, ParallelTensorShape> const &) const {
-  // TODO: more check on the input shape
-  return true;
+    std::pair<ParallelTensorShape, ParallelTensorShape> const &input) const {
+  return input.first.is_valid() && input.second.is_valid();
```

Comment:
We probably also want a check that the sizes of the two inputs are the same

Reply:
From my understanding of `ParallelTensorShape` this can be done with `==`. Just wanted to confirm that this is the correct way to check if two shapes are the same.

Reply:
Yeah that should be fine.